### PR TITLE
Tip hash metrics

### PR DESCRIPTION
**Pull request by newhoggy**
Salvaging this PR https://github.com/input-output-hk/cardano-node/pull/2006

Related to https://github.com/input-output-hk/cardano-node/issues/801

### CODE REVIEW

**Comment by intricate** [comment]
Thanks :grin:

File: cardano-node/src/Cardano/Tracing/Tracers.hs [hunk 1]
```diff
@@ -1482,23 +1504,30 @@ data ChainInformation = ChainInformation
     -- current chain.
   , fork :: Bool
     -- ^ Was this a fork.
+  , tipBlockHash :: HeaderHash blk
```
Comment (Jimbo4350) [inline cardano-node/src/Cardano/Tracing/Tracers.hs:1521]: I would argue lets not parameterize `ChainInformation` and instead have the following fields:
```
...
  , tipBlockHash :: Text
  , tipBlockParentHash :: Text
...
```
These are chain hashes that eventually get rendered anyways.

**Review by Jimbo4350 — DISMISSED** [review]

File: cardano-node/src/Cardano/Tracing/Tracers.hs (hunk 1 above)
Comment (newhoggy) [reply cardano-node/src/Cardano/Tracing/Tracers.hs]: Done!

**Review by erikd — APPROVED** [review]

File: cardano-node/src/Cardano/Tracing/Tracers.hs (hunk 1 above)
Comment (newhoggy) [reply cardano-node/src/Cardano/Tracing/Tracers.hs]: Done!

File: cardano-node/src/Cardano/Tracing/Tracers.hs [hunk 2]
```diff
@@ -1466,7 +1492,7 @@ traceInboundGovernorCountersMetrics (OnOff True) (Just ekgDirect) = ipgcTracer
 
 -- | get information about a chain fragment
 
-data ChainInformation = ChainInformation
+data ChainInformation blk = ChainInformation
```
Comment (Jimbo4350) [inline cardano-node/src/Cardano/Tracing/Tracers.hs:1495]: This is still parameterized

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]
LGTM! You should be able to squash your commit into the first one without issue

**Pull request merged**